utility/tests/TemplateUtil_unittest.cpp

```diff
@@ -25,18 +25,18 @@
 
 #include "utility/Macros.hpp"
 
-#include "glog/logging.h"
 #include "gtest/gtest.h"
 
 namespace quickstep {
 
 class SomeArgType {
  public:
-  explicit SomeArgType(std::string value) : value_(value) {
+  explicit SomeArgType(const std::string &value)
+      : value_(value) {
   }
 
-  SomeArgType(SomeArgType &&arg) {
-    value_ = std::move(arg.value_);
+  explicit SomeArgType(SomeArgType &&arg)
```

Comment:
Remove `explicit` for the move constructor to allow implicit construction from rvalue of `SomeArgType`.

Google `explicit move constructor`.